lib/tictactoe/game.rb

```diff
@@ -71,25 +72,49 @@ def check_winner()
       players = ["o", "x"]
       players.each do |symbol|
         if winning_combinations(symbol)
-          puts "Player using '#{symbol}' has won!"
-          return true
+          return true, symbol
         end
       end
+      return false
+    end
+
+    def output_wining_message(symbol)
+      @io.output(@messages.get_message("player_wins", symbol))
+    end
+  
+    def play_again_input
+      @io.output(@messages.get_message("play_again"))
+      input=@io.input
+      if input == "1"
+        @game = true
+      end
+    end
+
+    def play_again()
+      while @game
+        play
+      end
     end
 
     def play()
       draw()
-      while @board.is_not_full
-        if check_winner() != true
-          symbol = get_symbol()
-          position = get_position()
-          make_move(position, symbol)
-          draw()
-        else
-          return false
-        end
+      while @board.is_not_full && !check_winner()
```

Comment:
This could be hard to read for someone without knowledge of the game, could you turn this into a more descriptive method?